cmd/natc/natc.go

```diff
@@ -70,6 +71,17 @@ func main() {
 		log.Fatalf("site-id must be in the range [0, 65535]")
 	}
 
+	var ignoreDstTable *bart.Table[bool]
+	if *ignoreDstPfxStr != "" {
+		for _, s := range strings.Split(*ignoreDstPfxStr, ",") {
+			if s != "" {
+				if ignoreDstTable == nil {
+					ignoreDstTable = &bart.Table[bool]{}
+				}
+				ignoreDstTable.Insert(netip.MustParsePrefix(strings.TrimSpace(s)), true)
```

Comment:
It would be a much better UX to print an error rather than panic if parsing fails. Given the experimental status of this I guess it's not technically required but it would make me feel better.